package.json

```diff
@@ -76,6 +77,7 @@
     "npm": ">=8.15.0"
   },
   "devDependencies": {
+    "@11ty/eleventy": "^1.0.2",
     "@axe-core/playwright": "^4.5.2",
     "@commitlint/cli": "^17.3.0",
     "@commitlint/config-conventional": "^17.3.0",
```

Comment:
Just spotted a couple more Fastify dependencies (`@fastify/static`, `@fastify/view`) here that can be removed now:

Reply:
also removing nodemon